lib/host.js

```diff
@@ -251,14 +267,16 @@ HostConnectionPool.prototype._maybeCreatePool = function (callback) {
       return connections.length < self.coreConnectionsLength;
     },
     function iterator(next) {
-      var c = new Connection(self.address, self.protocolVersion, self.options);
-      //Relay the event idleRequestError
-      c.on('idleRequestError', function (err) {
-        //The pool will emit the event
-        self.emit('idleRequestError', err);
-      });
+      var c = self._createConnection();
```

Comment:
I wanted to make the first connection and, if it succeeds, try to open the rest in parallel. Then I remembered that it is not common to use more than 1 connection per host, as generally there are multiple (1 per core) nodejs processes running in the same machine...